### PR TITLE
feature/AboutUs - Added AboutUs component

### LINKED ISSUE #33: create about us component and add in home page
image
refer this image 
you can also refer https://girlcodeit.com/#about

### PR DESCRIPTION
**Pull request by VishalPatil18**
# Description

Created the AboutUs component and added it to the Home Page.

Fixed #33

## Type of change

Please check options that are relevant to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/CSS/AboutUs.module.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.AboutUs_Us__Embxm{
+    letter-spacing: 0;
+    color: #008dc8;
+    font-size: 50px;
+    font-weight: 500;
+}
+
+.AboutUs_AboutImage__11ku5{ margin-top:15% }
```
Comment (abdus) [inline src/CSS/AboutUs.module.css]: those class names ... did you manually append hashes(`3Gzv2` etc) after the name?

**Review by abdus — CHANGES_REQUESTED** [review]

File: src/CSS/AboutUs.module.css (hunk 1 above)
Comment (VishalPatil18) [reply src/CSS/AboutUs.module.css]: Not actually I saw it in the styles from the site mentioned in the repo i.e. https://girlcodeit.com. If you want me to remove those I can do it. Let me know what should be my next task.
Thanks!

Comment (abdus) [reply src/CSS/AboutUs.module.css]: Okay. You can remove those extra hashes. It will be generated automatically at build time by webpack.

Comment (VishalPatil18) [reply src/CSS/AboutUs.module.css]: Sure will do that ASAP.

Comment (VishalPatil18) [reply src/CSS/AboutUs.module.css]: @abdus I've made the required changes. You can review them now.
Thanks!

**Review by abdus — APPROVED** [review]

**Pull request merged**